terraform/main.tf

```diff
@@ -1,41 +1,51 @@
 
-locals {
-  aws-region-2 = "us-west-2"
-}
-
 // Create S3 bucket for uploading binary
 resource "aws_s3_bucket" "idp-id-broker-search" {
   bucket        = "${var.app_name}-${var.aws_region}"
-  acl           = "public-read"
   force_destroy = true
 
-  versioning {
-    enabled = true
-  }
-
   tags = {
     app_name = var.app_name
     app_env  = var.app_env
   }
 }
 
+resource "aws_s3_bucket_acl" "idp-id-broker-search" {
+  bucket = aws_s3_bucket.idp-id-broker-search
```

Comment:
Should this and the next `bucket` value match the one up on line 4 (or vice-versa)?

Reply:
@Baggerone I think this value is simply referencing the resource type and name above (i.e. this is saying "use the 'aws_s3_bucket' resource in this terraform file that's labeled 'idp-id-broker-search'"). I think the actual _name_ of the bucket (on S3) is the string defined in that resource up on line 4.

All that to say, I think this is correct as-is, but @briskt can confirm for sure.

Reply:
Oh, right.

Reply:
Actually, @Baggerone was right. But typically, I use the `id` attribute of the created bucket rather than copy the `bucket` argument value. So I just need to add `.id` on the end of these lines.